File: 2390. Removing Stars From a String/Solution.js
/**
 * @param {string} s
 * @return {string}
 */
var removeStars = function(s) {
    stack = []
    for(let char of s){
        if(char == '*'){
             stack.pop();
        }
        else{
            stack.push(char)
        }
            
    }
    return stack.join('');
};

File: 2390. Removing Stars From a String/Solution.py
class Solution:
    def removeStars(self, s: str) -> str:
        stack = []

        for i in range(len(s)):
            if s[i] == '*':
                stack.pop()
            else:
                stack.append(s[i])
        return ''.join(stack)
    
s = Solution();
print(s.removeStars("leet**cod*e"))